geoviews/tests/conftest.py

```diff
@@ -1,5 +1,7 @@
 from contextlib import suppress
 
+from holoviews.tests.conftest import bokeh_backend, port, serve_hv  # noqa: F401
+
```

Comment:
```suggestion
from holoviews.tests.conftest import bokeh_backend, port, serve_hv  # noqa: F401

```

Reply:
Was this accidental?

Reply:
Yes (I updated it just now). I don't think these changes are related to this PR.

Reply:
Ah I see :) leaked from the other PR.